gridded_etl_tools/dataset_manager.py

```diff
@@ -58,6 +58,8 @@ def __init__(
         allow_overwrite=False,
         ipfs_host="http://127.0.0.1:5001",
         dask_dashboard_address: str = "127.0.0.1:8787",
+        dask_worker_memory_target: float = 0.65,
```

Comment:
We should really start documenting these params in the docstring but understand if you want to leave it for a future PR